src/BattleCalculator-Freewar/BattleCalculator-Freewar.user.js

```diff
@@ -8,10 +8,40 @@
 // @grant       none
 // ==/UserScript==
 /* global $ */
+    var targetNode = document.querySelector('.listusersrow, .listusersrow table tr td');
+    if (targetNode) {
+      var observerOptions = {
+        childList: true,
+        subtree: true
+      };
 
+      var observer = new MutationObserver(function(mutationsList) {
+        routine();
+      });
+
+      observer.observe(targetNode, observerOptions);
+    } else {
+      //console.error("Target node with class 'listusersrow' or 'listusersrow table tr td' not found!");
+    }
```

Comment:
leftover code from debugging, might want to remove it instead of commenting out

Reply:
removed